src/openqdc/datasets/base.py

```diff
@@ -81,21 +81,27 @@ class BaseDataset(torch.utils.data.Dataset):
     __fn_forces__ = lambda x: x
 
     def __init__(
-        self, energy_unit: Optional[str] = None, distance_unit: Optional[str] = None, cache_dir: Optional[str] = None
+        self,
+        energy_unit: Optional[str] = None,
+        distance_unit: Optional[str] = None,
+        overwrite_local_cache: bool = False,
+        cache_dir: Optional[str] = None,
     ) -> None:
         set_cache_dir(cache_dir)
         self.data = None
         self._set_units(energy_unit, distance_unit)
         if not self.is_preprocessed():
-            entries = self.read_raw_entries()
-            res = self.collate_list(entries)
-            self.save_preprocess(res)
-        self.read_preprocess()
-        self.__isolated_atom_energies__ = (
-            [IsolatedAtomEnergyFactory.get(en_method) for en_method in self.__energy_methods__]
-            if self.__energy_methods__
-            else None
-        )
+            logger.info("This dataset not available. Please open an issue on Github for the team to look into it.")
+            # entries = self.read_raw_entries()
+            # res = self.collate_list(entries)
+            # self.save_preprocess(res)
```

Comment:
Why we removed this and wrapped it into a function but we are not calling it?

Reply:
We are calling the function in to [preprocessing file](https://github.com/OpenDrugDiscovery/openQDC/blob/42e8db8bed1b32c5ae2f09b95df6075a608332fb/src/openqdc/utils/preprocess.py#L36). The lambda user of the library don't have access to the raw files and don't need to call the function so this easily prevent unwanted calls to the function. We can even remove the function and put it into a class that is only accessible in the preprocessing file.